src/vueWrapper.ts

```diff
@@ -176,30 +160,7 @@ export class VueWrapper<T extends ComponentPublicInstance>
     return createWrapperError('VueWrapper')
   }
 
-  getComponent<T extends ComponentPublicInstance>(
-    selector: FindComponentSelector | (new () => T)
-  ): Omit<VueWrapper<T>, 'exists'> {
-    const result = this.findComponent(selector)
-
-    if (result instanceof VueWrapper) {
-      return result as VueWrapper<T>
-    }
-
-    let message = 'Unable to get '
-    if (typeof selector === 'string') {
-      message += `component with selector ${selector}`
-    } else if ('name' in selector) {
-      message += `component with name ${selector.name}`
-    } else if ('ref' in selector) {
-      message += `component with ref ${selector.ref}`
-    } else {
-      message += 'specified component'
-    }
-    message += ` within: ${this.html()}`
-    throw new Error(message)
-  }
-
-  findAllComponents(selector: FindAllComponentsSelector): VueWrapper<T>[] {
+  findAllComponents(selector: FindAllComponentsSelector): VueWrapper<any>[] {
```

Comment:
Replaced `VueWrapper<T>` with `VueWrapper<any>`. While this typing could be definitely improved (I hate `any` anywhere :rofl: ) it is at least "not incorrect" - `<T>` here was a type of current instance, so if you've done 
```js
const wrapper = mount(Hello);
const cmps = wrapper.findAll(Foo);
```
cmps was wrongly typed to `VueWrapper<Hello>[]` instead of `VueWrapper<Foo>[]`

Reply:
Oh good catch! But `any` feels definitely wrong.
Would you mind opening a dedicated PR to fix this properly (or we can file an issue)?
I think it would be better to at least let developers provide the type like `wrapper.findAllComponents<Foo>(Foo)` to get `Array<VueWrapper<Foo>>`. Or even better infer it from the selector if possible (but I'm not sure it's doable in all cases, as the developer may specify `{ name: 'Foo' }` as a selector).
What do you think?

Reply:
Definitely agree. I've opened separate issue https://github.com/vuejs/vue-test-utils-next/issues/899 because I can't estimate effort to fix this properly, yet current PR has value - it is LESS wrong than previous version.